experimental/OscarDB/src/OscarDB.jl

```diff
@@ -0,0 +1,208 @@
+module OscarDB
+
+using ..Oscar
+using ..Oscar.Serialization
+
+import Oscar.Serialization: load_object, save_object, type_params
+
+# Transitivesimplicialcomplex imports
+import Oscar:
+  simplicial_complex,
+  dim,
+  f_vector,
+  automorphism_group,
+  homology,
+  betti_numbers,
+  n_vertices
+
+# for ca certificates
+using NetworkOptions, URIs
```

Comment:
```suggestion
import NetworkOptions
import URIs
```
this would be better as it does not pollute your namespace, and you qualify the calls into these packages anyway (at least for NetworkOptions). For URIs, there are only a handfull of calls as well; so it would be clearer for these to become qualified as well.